lib/gap_intelligence/client/downloads.rb

```diff
@@ -28,5 +28,18 @@ def downloads(owner_id, params = {}, options = {}, &block)
       default_option(options, :record_class, Download)
       perform_request(:get, 'downloads', options.merge(params: params.merge(owner_id: owner_id)), &block)
     end
+
+    # Deletes download
+    #
+    # @param owner_id [String,Integer] owner id of the download
+    # @param download_ids [Integer] id of the download
+    # @param options [Hash] options the options to make the request with
+    # @yield [request] The Faraday request
+    # @return [Download] the created download
```

Comment:
update: just noticed in https://github.com/GapIntelligence/gapi/pull/1068/files that endpoint actually returns `render json: { data: { status: 'OK' } }`, not actual Download instance